test/test-debuglet.ts

```diff
@@ -315,6 +316,49 @@ describe('Debuglet', function() {
       assert.deepEqual(mergedConfig, compareConfig);
     });
 
+    it('should elaborate on inspector warning on 32 bit', function(done) {
+      let logText = '';
+      const oldLog = console.log;
+      console.log = function(s: string) {
```

Comment:
I would suggest setting `console.log` in `beforeEach` and restoring `console.log` if `afterEach`.  Otherwise, if the test fails, the code for restoring `console.log` won't get run, which could influence any tests that follow.

Reply:
Done.